components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js

```diff
@@ -1,31 +1,28 @@
 import { useEffect, useState, useCallback } from 'react';
 import axios from 'axios';
-import { useDispatch, useSelector } from 'react-redux';
 import { AccountVerificationFormResumeInBackgroundModal } from './AccountVerificationFormResumeInBackgroundModal';
 import { useAccountVerificationForm } from './AccountVerificationFormProvider';
 import { useTernaryState } from '@/utils/useTernaryState';
-import { fetchUserTransactions, userTransactionsLoading } from '@/store/actions/userTransactionsActions';
 import { Button } from '@/components/Button';
 import { CircularProgressBar } from '@/components/CircularProgressBar';
+import { useTransactionsDataContext } from '@/components/store/context/transactionContext';
 
 export function AccountVerificationFormStep3LoadingSteps() {
   const [isResumeModalOpen, openResumeModal, closeResumeModal] = useTernaryState(false);
   const [progressBarValue, setProgressBarValue] = useState(0);
   const [progressInterval, setProgressInterval] = useState(null);
-  const [transactionLoadingDispatched, setTransactionLoadingDispatched] = useState(false);
-
-  useDispatch(userTransactionsLoading());
-  const { isCompleted, transactionsError } = useSelector(state => state.userTransactions);
+  const transactionContext = useTransactionsDataContext();
+  const isCompleted = transactionContext.state.isCompleted;
+  const transactionsError = transactionContext.state.transactionsError;
```

Comment:
Can you update as below or similar ? 
_const { isCompleted, transactionsError } = transactionContext.state;_

Reply:
updated to a similar structure